react-frontend/src/components/CaseStudies/casestudies.js

```diff
@@ -19,7 +20,9 @@ const CaseStudies = () => {
             </div>
           </div>
         </Route>
-        <Route path={path + '/:caseStudyId'} component={Template} />
+        <div className="pageContainer">
```

Comment:
Is there a reason this is called a pageContainer?  Is the entire case studies page contained in a single <Route> object?  I'm mostly trying to understand the page structure here.